jetson/nav/README.md

```diff
@@ -81,4 +81,71 @@ This is just a boolean hidden as a type called AutonState and it tells us if Aut
 Contains functions used commonly throughout auton code. 
 
 
+---
+
+<!----------------------------- LCMs ----------------------------->
+## LCM Channels Publishing/Subscribed To 
+**Target [subscriber]** \
+Messages: [ AutonState.lcm ](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/AutonState.lcm) “/auton_state” \
+Publishers: jetson/teleop \
+Subscribers: jetson/nav
+
+**Course [subscriber]** \
+Messages: [ Course.lcm ](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/Course.lcm) “/course” \
+Publishers: simulators/nav, base_station \
+Subscribers: jetson/nav
+
+**Obstacle [subscriber]** \
+Messages: [ Obstacle.lcm ](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/Obstacle.lcm) “/obstacle” \
+Publishers: jetson/percep \
+Subscribers: jetson/nav
+
+**Odometry [subscriber]** \
+Messages: [ Odometry.lcm ](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/Odometry.lcm) “/odometry” \
+Publishers: jetson/gps \
+Subscribers: jetson/nav
+
+**Target List [subscriber]** \
+Messages: [ TargetList.lcm ](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/TargetList.lcm) “/target_list” \
+Publishers: jetson/percep \
+Subscribers: jetson/nav
+
+**ZED Gimbal Data [subscriber]** \
+Messages: [ ZedGimalPosition.lcm ](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/TargetList.lcm) “/zed_gimbal_data” \
+Publishers: simulators/nav \
+Subscribers: jetson/nav
+
```

Comment:
do you guys publish to zed_gimbal_cmd?

Reply:
Yes, but that's not merged into main yet, so I had left it out. Should I include it in this iteration?

Reply:
maybe add it as a todo in the readme, or add it but put a note underneath saying not implemented yet

Reply:
ok im going to approve this to avoid re-approval, but I meant add it into the #TODO section of a readme like in[ the readme guide ](https://github.com/umrover/embedded-testbench/wiki/The-README-Guide#todo)with a description of what the todo is (i.e add the publishing to zed gimbal cmd in the state machine)